package.json

```diff
@@ -27,9 +45,12 @@
     "ethers": "^5.7.0"
   },
   "scripts": {
-    "tests": "mocha -r ts-node/register --bail tests/setup.test.ts tests/unit/*.test.ts tests/integration/*.test.ts",
+    "test": "mocha -r ts-node/register tests/**/*.test.ts",
```

Comment:
`tests/setup.test.ts` needs to run before all other tests because it sets the test environment. The --bail flag is set to stop other tests from running if the setup wasn't successful.